repo2docker/buildpacks/python/requirements.frozen.txt

```diff
@@ -30,13 +30,13 @@ ptyprocess==0.5.2
 Pygments==2.2.0
 python-dateutil==2.6.1
 pyzmq==17.0.0
-Send2Trash==1.4.2
+Send2Trash==1.5.0
 simplegeneric==0.8.1
 six==1.11.0
 terminado==0.8.1
 testpath==0.3.1
-tornado==4.5.3
+tornado==5.0
```

Comment:
This looks like we should briefly think about it. Especially as the conda environments will remain at 4.5. This makes me think this is not what should be happening -> we should remain on 4.5 here as well. @minrk knows about tornado 5 vs 4.

Reply:
I could force pin tornado at the top level too.